src/server_manager/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "outline-manager",
   "productName": "Outline Manager",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Don't we have some new features? I think this should be 1.5.0

Reply:
Good catch, corrected now.